solr/CHANGES.txt

```diff
@@ -100,6 +100,8 @@ Other Changes
 
 * SOLR-17279: Introduce SecurityJson.java file to Test Framework to consolidate setting up authentication in tests. (Rudy Seitz via Eric Pugh)
 
+* SOLR-15582: Remove deprecated APIs in solrj Tuple class. (Christine Poerschke via Eric Pugh)
+
```

Comment:
Actually the removal already happened (for both `main` and `branch_9x`) -- https://github.com/apache/solr/commit/70fbfb6688741c81602a0fcfc8b1b0877d9fdeda -- and hence there were 'only' Javadoc changes left here then. How about we remove the CHANGES.txt entry here then to avoid confusion?